odoo/addons/base/tests/test_views.py

```diff
@@ -3090,3 +3089,22 @@ def test_views(self):
                 _logger.info('checked %s/%s views', index, len(views))
             with self.subTest(name=view.name):
                 view._check_xml()
+
+@common.tagged('post_install', '-at_install', '-standard', 'render_all_views')
```

Comment:
The `--test-file` argv ignores the `-standard` tag. How could we make `--test-file` skips this test or is there a `--test-tags` equivalent that we could use to run all tests classes in a module ?